packages/myst-to-typst/src/index.ts

```diff
@@ -341,6 +348,12 @@ const handlers: Record<string, Handler> = {
   //     );
   //   }
   // },
+  div(node, state) {
+    state.renderChildren(node, 1);
+  },
+  span(node, state) {
+    state.renderChildren(node, 0, { trimEnd: false });
```

Comment:
I was wondering the justification for `trimEnd: false` but `trailingNewLines: 0`... I think, though, this implies we are in a paragraph (so no new lines) and the intended trailing whitespace is just spaces (which we want to keep).

This feels slightly risky - we have less control over the `typst` output in cases where we get strange `myst` - but then again, adding something like `trailingSpaces` feels maybe too micromanage-y.

Probably fine as is... I suspect the use-cases for `span` are pretty narrow.

Reply:
Yeah, I think that the short-term way to go is to have a `renderInline` that does the children, without the line ending stuff.